### PR TITLE
Übersetzung hinzugefügt

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: locale/justoverclock-flarum-ext-welcomebox [hunk 1]
```diff
@@ -0,0 +1,8 @@
+  forum:
```
Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-welcomebox:1]: First line same as with other file, must contain "flarum-ext-welcomebox" and filename must also match that as "flarum-ext-welcomebox.yml".

File: locale/justoverclock-flarum-ext-welcomebox [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    wback: Willkommen zurück,
+    npost: => core.ref.posts
+    discussion: => core.ref.discussions
+    welcomeguest: Willkommen Gast
+    notregistered: Du bist aktuell nicht im Forum registriert.
```
Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-welcomebox:6]: 5 translations are missing here.

**Review by Kakifrucht — CHANGES_REQUESTED** [review]

File: locale/justoverclock-flarum-ext-welcomebox (hunk 2 above)
Comment (Justman10000) [reply locale/justoverclock-flarum-ext-welcomebox:6]: Where?

Comment (Kakifrucht) [reply locale/justoverclock-flarum-ext-welcomebox:6]: These ones: https://github.com/justoverclockl/flarum-ext-welcomebox/blob/main/resources/locale/en.yml#L8

File: locale/justoverclock-flarum-ext-welcomebox [hunk 3]
```diff
@@ -0,0 +1,8 @@
+  justoverclock-flarum-ext-welcomebox:
```
Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-welcomebox:1]: First line is the same as in the original file, so only flarum-ext-welcomebox, filename would be flarum-ext-welcomebox.yml, and we still need the forum parent key.

**Pull request merged**